app.js

```diff
@@ -22,6 +23,7 @@ app.use(express.static(path.join(__dirname, 'public')));
 
 app.use('/', indexRouter);
 app.use('/users', usersRouter);
+app.use('/authors', authorsRouter);
```

Comment:
czy czasem nie miały się zaczynać ścieżki od /api? (/api/authors) w tym przypadku

Reply:
kwestia ustaleń, jeżeli chcecie zaczynać od API to napiszcie to zmienie

Reply:
tako jest na liście i nikt nie zgłosił do tego zastrzeżeń

Reply:
poprawione